lib/taskana-cdi-example/src/main/java/pro/taskana/example/ExampleBootstrap.java

```diff
@@ -1,57 +1,63 @@
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.enterprise.event.Observes;
+import jakarta.enterprise.event.Startup;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pro.taskana.classification.api.models.Classification;
 import pro.taskana.task.api.models.Task;
 import pro.taskana.task.internal.models.ObjectReferenceImpl;
-import pro.taskana.workbasket.api.exceptions.NotAuthorizedOnWorkbasketException;
-import pro.taskana.workbasket.api.exceptions.WorkbasketNotFoundException;
+import pro.taskana.workbasket.api.WorkbasketType;
+import pro.taskana.workbasket.api.models.Workbasket;
 
 /** Example Bootstrap Application. */
 @ApplicationScoped
 public class ExampleBootstrap {
 
+  private static final Logger LOGGER = LoggerFactory.getLogger(ExampleBootstrap.class);
+
+  private static final String CDIDOMAIN = "CDIDOMAIN";
+  private static final String CLASSIFICATION_TYPE = "T1";
+
   @EJB private TaskanaEjb taskanaEjb;
 
-  @PostConstruct
-  public void init(@Observes @Initialized(ApplicationScoped.class) Object init)
-      throws TaskNotFoundException,
-          WorkbasketNotFoundException,
-          ClassificationNotFoundException,
-          InvalidOwnerException,
-          TaskAlreadyExistException,
-          InvalidArgumentException,
-          AttachmentPersistenceException,
-          ObjectReferencePersistenceException,
-          NotAuthorizedOnWorkbasketException,
-          InvalidTaskStateException {
-    System.out.println("---------------------------> Start App");
+  /**
+   * The parameter `@Observes Startup` makes sure that the dependency injection framework calls this
+   * method on system startup. And to do that, it needs to call `@PostConstruct start()` first.
+   *
+   * @param startup just the startup event
+   * @throws Exception in case of task creation fails
+   */
+  public void init(@Observes Startup startup) throws Exception {
+    LOGGER.info("---------------------------> Start App -- {}", startup);
+    Workbasket workbasket = taskanaEjb.getWorkbasketService().newWorkbasket("KEY", CDIDOMAIN);
+    workbasket.setName("wb");
+    workbasket.setType(WorkbasketType.PERSONAL);
+    workbasket = taskanaEjb.getWorkbasketService().createWorkbasket(workbasket);
+    Classification classification =
+        taskanaEjb
+            .getClassificationService()
+            .newClassification("TEST", CDIDOMAIN, CLASSIFICATION_TYPE);
+    taskanaEjb.getClassificationService().createClassification(classification);
     ObjectReferenceImpl objRef = new ObjectReferenceImpl();
     objRef.setCompany("aCompany");
     objRef.setSystem("aSystem");
     objRef.setSystemInstance("anInstance");
     objRef.setType("aType");
     objRef.setValue("aValue");
-    Task task = taskanaEjb.getTaskService().newTask(null);
+    Task task = taskanaEjb.getTaskService().newTask(workbasket.getId());
+    task.setClassificationKey(classification.getKey());
+    task.setName("BootstrapTask");
```

Comment:
Why do you set the Task name here? Or is it for clearness?

Reply:
For better Testing.

In the Testcase I search for exactly this Taskname